src/main/java/seedu/address/model/person/Person.java

```diff
@@ -93,12 +123,14 @@ public boolean equals(Object other) {
                 && otherPerson.getEmail().equals(getEmail())
                 && otherPerson.getAddress().equals(getAddress())
                 && otherPerson.getTags().equals(getTags());
```

Comment:
Unrelated to this PR, but do you guys want to break this long boolean expression into smaller ones?

Reply:
Nope, to me all these boolean expression is intuitive and understandable that each attributes of a person needs to be equal. It is unlike checking two or more completely different booleans checks.